Src/Test.UnitTests.Sarif.PatternMatcher.Cli/ProgramTests.cs

```diff
@@ -22,6 +23,26 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
 {
     public class ProgramTests
     {
+        [Fact]
+        public void Program_ResponseFileWorksWithNoExplicitlyAllocatedFileSystem()
```

Comment:
>Program_ResponseFileWorksWithNoExplicitlyAllocatedFileSystem [](http://example.com/codeflow?start=20&length=60)

Funny corner case. This functionality is highly tested, but all tests pass a FileSystem mock object meaning that data was always non-null. In production, we missed a null deref when that mock object wasn't populated. :) 

This happens all the time, good testing still leaves little gaps, nooks and crannies in the actual production/runtime code paths...